web-comp.js

```diff
@@ -0,0 +1,204 @@
+const catalogProducts = [
```

Comment:
Do not use abbreviations!!!!!!!
Also rename this file using a more appropriate name.

"web-comp" is too generic.

Reply:
fixed